noxfile.py

```diff
@@ -23,6 +25,17 @@ def build_docs(session: nox.Session) -> None:
             ".",
             f"{envbindir}/../tmp/html",
         )
+    # For CI testing if documentation builds
+    else:
+        session.run(
+            "sphinx-build",
+            "-b",
+            "html",
+            "-W",
+            "--keep-going",
+            ".",
+            f"{envbindir}/../tmp/html",
+        )
```

Comment:
(nit) I know we use this in PyBaMM too, but I usually find it better to find the built files if they are placed in the `docs/build/html` folder itself, and it makes working without `nox` easier because one doesn't need to go into the `.nox/` folders